sanic/worker/manager.py

```diff
@@ -408,6 +408,23 @@ def remove_worker(self, worker: Worker) -> None:
         logger.info("Removed worker %s", worker.ident)
         del worker
 
+    def terminate_durable_worker(self, ident: str) -> None:
+        """Terminate a durable worker and its processes.
+
+        Args:
+            ident (str): The name of the durable process to terminate.
+        """
+        worker = self.durable.get(ident)
+        if not worker:
+            error_logger.error(
+                "Durable worker termination failed (%s not found).", worker
```

Comment:
Should be `, ident`; not `, worker`. That seems to be what's making the test fail.